.github/workflows/test.yaml

```diff
@@ -38,9 +38,16 @@ jobs:
           go-version: ${{ inputs.go-version }}
           cache: true
 
+      - name: Install Firebase Emulator Suite
+        run: |
+          npm install -g firebase-tools@12.0.1
+
       - name: Test
         env:
           IGN_DB_ADDRESS: "127.0.0.1:${{ job.services.mysql.ports[3306] }}"
+          GOOGLE_CLOUD_PROJECT: "test-project"
+          FIRESTORE_EMULATOR_HOST: "localhost:8080"
         run: |
+          firebase emulators:start --project test-project
```

Comment:
This is a blocking operation and prevents the workflow from running.

Reply:
Can you send it to the background?

```bash
firebase emulators:start --project test-project &
```